src/components/Sidebar/Sidebar.minors.tsx

```diff
@@ -42,7 +56,7 @@ const ItemStyled = styled(Button)<{
   text-align: left;
   padding: 0.25rem 0.25rem 0.25rem 0.75rem;
   justify-content: start;
-  ${({ isActive, activeStyles }) => (isActive ? activeStyles : '')};
+  ${({ isActive, activeStyles }) => (isActive && activeStyles ? activeStyles : '')};
```

Comment:
related to [this change](https://github.com/vimeo/iris/pull/271/files#r1211924697)